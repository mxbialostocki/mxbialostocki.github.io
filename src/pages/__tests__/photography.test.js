import React from "react"
import ShallowRenderer from "react-test-renderer/shallow"

import Photography from "../photography"

describe("Photography Page", () => {
  const renderer = new ShallowRenderer()
  it("renders correctly", () => {
    const tree = renderer.render(
    <Photography />
    )
    expect(tree).toMatchSnapshot()
  })
})
