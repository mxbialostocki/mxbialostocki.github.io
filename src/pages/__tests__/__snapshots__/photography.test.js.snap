// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photography Page renders correctly 1`] = `
<React.Fragment>
  <Layout>
    <SEO
      description=""
      lang="en"
      meta={Array []}
      title="Photography"
    />
    <WithStyles(ForwardRef(Grid))
      container={true}
      direction="column"
      spacing={4}
    >
      <WithStyles(ForwardRef(Grid))
        item={true}
      >
        <WithStyles(ForwardRef(Typography))
          variant="h5"
        >
          I apply a developed, personal method to my portraiture and offer event photography at fair prices. 
          <Link
            to="/contact"
          >
            <span
              className="makeStyles-primary-1"
            >
              Contact me
            </span>
          </Link>
           to start a conversation about your needs or book in a session.
        </WithStyles(ForwardRef(Typography))>
      </WithStyles(ForwardRef(Grid))>
    </WithStyles(ForwardRef(Grid))>
    <PhotographyGallery />
  </Layout>
</React.Fragment>
`;
